const ftWithError = () => {
    throw new Error('Какая та ощибка')
}

ftWithError()
console.log('Continue..')